demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -248,8 +248,13 @@ def generate_single_command_section(cmd: dict, example_dict: dict, command_permi
             if not arg.get('description'):
                 errors.append(
                     'Error! You are missing description in input {} of command {}'.format(arg['name'], cmd['name']))
+            argument_description = f'{arg.get("description")} The optional values are {arg.get("predefined")}.' \
```

Comment:
please work with @kirbles19 on the text

Reply:
Possible values are: {arg.get("predefined")}. Default is {arg.get("defaultValue")}.

Reply:
@Shellyber note the value for the possible values need to be comma separated without the squared brackets, so you will need to use `join`

Reply:
@Itay4 Done. Please approve also